src/zen_octet.c

```diff
@@ -1976,6 +1976,7 @@ int luaopen_octet(lua_State *L) {
 		{"to_array",  to_array},
 		{"to_octet",  to_octet},
 		{"to_bin",    to_bin},
+		{"zcash_topoint", zcash_topoint},
```

Comment:
This should be implemented as `ECP.from_zcash(octet)` and `ECP2.from_zcash(octet)`

I am aware this doesn't only concerns this PR since zcash point encoding was introduced in bbs, but perhaps now is a good time to normalize this interface. It will make also much more readable code where :zcash_topoint() is being used, making it explicit that an ECP is returned, when the method is `ECP.from_zcash(input_octet)`